core/src/main/java/com/googlecode/psiprobe/tools/url/UrlParser.java

```diff
@@ -34,6 +34,13 @@
 
   /**
    * Instantiates a new url parser.
+   */
+  public UrlParser() {
```

Comment:
This was purposefully non-public. If you ever instantiate this class with this constructor, it's going to spit out a bunch of blank values.